### PR TITLE
Improve readability of pkgdb.lib.vcs_acls by using proper variable names

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/pkgdb2/issues/165

### CODE REVIEW

File: pkgdb2/lib/__init__.py [hunk 1]
```diff
@@ -2038,75 +2038,75 @@ def vcs_acls(session, eol=False, oformat='text', skip_pp=None):
...
+            if username and username.startswith('group::'):
+                    group = username.replace('group::', '')
             else:
-                user = pkg[1]
+                user = username
 
-            if pkg[0] not in output:
-                output[pkg[0].encode('utf-8')] = {}
+            if pkgname not in output:
+                output[pkgname.encode('utf-8')] = {}
```
Comment (ralphbean) [inline pkgdb2/lib/__init__.py]: Why 'utf-8' here?  Would it make sense to keep it unicode while in this loop and then finally convert to byte strings at the very end?

**Comment by ralphbean** [comment]
Okay, another suggestion.  This function contains two *giant* pieces on either side of ``if oformat == 'json':``.  Can you break those code-blocks into two separate functions?

```python
if oformat == 'json':
    return _do_vcs_acls_json(arguments)
else:
    return _do_vcs_acls_the_other_format(arguments)
```

Then the docstring of each of those utility methods could describe what they're
for, what the different formats are like, what uses them, etc..

**Comment by pypingou** [comment]
sure can do

File: pkgdb2/lib/__init__.py (hunk 1 above)
Comment (pypingou) [inline pkgdb2/lib/__init__.py]: I think that's a left over from when I tried to build the JSON by hand using a dict structure.

File: pkgdb2/lib/__init__.py [hunk 2]
```diff
@@ -2025,6 +2025,134 @@ def bugzilla(session, name=None):
...
+
+    The output dict is something like:
+
+    {
+      pkg1: {
+        branch1: {
+          name: pkg1,
+          branch: branch1,
+          user: [user1, user2],
+          group: [group1, group2]
```
Comment (ralphbean) [inline pkgdb2/lib/__init__.py]: This looks like it should be ``groups`` not ``group`` and ``people`` not ``user`` (from reading the code below).

Comment (pypingou) [inline pkgdb2/lib/__init__.py]: No no, the key for groups is singular, see https://github.com/fedora-infra/pkgdb2/pull/166/files#diff-87909695400e927cca78825103e7f028R2139 and above :)

Comment (ralphbean) [inline pkgdb2/lib/__init__.py]: No, no.  See for ``_json`` it is plural https://github.com/fedora-infra/pkgdb2/pull/166/files#diff-87909695400e927cca78825103e7f028R2075 but for ``_text`` it is singular: https://github.com/fedora-infra/pkgdb2/pull/166/files#diff-87909695400e927cca78825103e7f028R2139

Comment (pypingou) [inline pkgdb2/lib/__init__.py]: Oh sorry, yes I was looking at the wrong method

**Comment by ralphbean** [comment]
:+1:!

**Comment by pypingou** [comment]
Thanks @ralphbean

**Pull request merged**